db/seeds.d/020-permissions_list.rb

```diff
@@ -99,6 +99,7 @@ def permissions
         ['Parameter', 'create_params'],
         ['Parameter', 'edit_params'],
         ['Parameter', 'destroy_params'],
+        ['LookupValue', 'create_lookup_values'],
```

Comment:
Can we move it, so the alphabetical order is preserved?

And since we're at it, maybe it's worth to add all the CRUD permissions?